### PR TITLE
EOS-20262-Monitor CortxMessageBusServer also restructured coverage pkg.py installation. 

### PR DESCRIPTION
**Pull request by sumedhak27**
Signed-off-by: Sumedh A. Kulkarni <sumedh.a.kulkarni@seagate.com>

## Problem Statement:

Control and Monitoring of cortx_message_bus.service should be enabled. 
coverage==5.5 pip pkg should be installed with sspl-test rpm and validated.

## Solution Design:
Made required code changes.

## Coding:

* [x] Did you follow coding standard and verified it with flake8? 

* [ ] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [x] Sanity/Self tests are updated (if applicable)?

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [x] Did you add Jira number to the PR?

* [x] DCO and cla-signed?

### CODE REVIEW

**Review by sandeepsgt — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml [hunk 1]
```diff
@@ -168,6 +168,7 @@ SERVICEMONITOR:
       - scsi-network-relay.service
       - kafka.service
       - kafka-zookeeper.service
+      - cortx_message_bus.service
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml:171]: THis is CORTX service, which will be monitored by HA (pcs). Why the exception for this service, to be monitored by SSPL ? @rahul-tripathi-git 
cc @ajaysrivas

File: low-level/files/opt/seagate/sspl/setup/sspl_test.py [hunk 2]
```diff
@@ -55,7 +55,8 @@ def validate(self):
             }
         # python 3rd party package dependency
         pip3_3ps_packages_test = {
-            "Flask": "1.1.1"
+            "Flask": "1.1.1",
+            "coverage": "5.5"
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/setup/sspl_test.py:59]: Merging two different unrelated changes in one PR !?

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml (hunk 1 above)
Comment (cdeshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml:171]: As per Ajay's reply on jira, this should happen through pcs. @rahul-tripathi-git , sync with HA team. SSPL will wait till monday EOD to conclude on this. May not merge in given sprint

**Comment by cdeshmukh** [comment]
@rahul-tripathi-git @sachinpunadikar do we need this PR still. Would like to close this, have been waiting for reply for long now

**Comment by sachinpunadikar** [comment]
@cdeshmukh , apologies for delay.
Please close this, as Rest server monitoring will be done by HA,
